corehq/apps/app_manager/models.py

```diff
@@ -3684,6 +3689,7 @@ class ShadowModule(ModuleBase, ModuleDetailsMixin):
     source_module_id = StringProperty()
     forms = []
     excluded_form_ids = SchemaListProperty()
+    form_session_endpoint_mapping = SchemaListProperty(ShadowFormMapping)
```

Comment:
nit: I like just `form_session_endpoints` here, as the stored value is a list (one per form), not a dict.

Reply:
Yeah. I was not sold on the mapping part either. Will change the name.